src/FakeClient/FakeCaptureEventProcessor.h

```diff
@@ -10,43 +10,79 @@
 
 namespace orbit_fake_client {
 
-// This implementation of CaptureEventProcessor simply discards all the events it receives, but it
-// keeps track of their number and total size and then outputs these statistics to file.
+// This implementation of CaptureEventProcessor mostly discard all events it receives, except for:
+// - keeping track of their number and total size, and writing these statistics to file;
+// - keeping track of the calls to the frame boundary function, and possibly writing the average
+//   frame time to file;
 class FakeCaptureEventProcessor : public orbit_capture_client::CaptureEventProcessor {
  public:
   void ProcessEvent(const orbit_grpc_protos::ClientCaptureEvent& event) override {
     ++event_count_;
     byte_count_ += event.ByteSizeLong();
+
+    if (event.has_function_call()) {
+      const orbit_grpc_protos::FunctionCall& function_call = event.function_call();
+      if (function_call.function_id() == kFrameBoundaryFunctionId) {
+        uint64_t start_timestamp_ns =
+            function_call.end_timestamp_ns() - function_call.duration_ns();
+        ++frame_boundary_count_;
```

Comment:
So you are basically counting the frames and then compute (max_timestamp - min_timestamp)/#frames? I have the feeling that you should add a brief comment here.

Reply:
Exactly. Added comment.